src/blib2to3/pgen2/conv.py

```diff
@@ -67,12 +67,10 @@ def parse_graminit_h(self, filename):
             return False
         self.symbol2number = {}
         self.number2symbol = {}
-        lineno = 0
-        for line in f:
-            lineno += 1
+        for lineno, line in enumerate(f, start=1):
             mo = re.match(r"^#define\s+(\w+)\s+(\d+)$", line)
             if not mo and line.strip():
-                print("%s(%s): can't parse %s" % (filename, lineno, line.strip()))
+                print("%s(%s): can't parse %s" % (filename, lineno+1, line.strip()))
```

Comment:
You need either this or the start=1 but not both.

Reply:
Yes, you are right!